useScroll/useScroll.js

```diff
@@ -223,10 +223,10 @@ const useThemeScroll = (props, instances) => {
 			contextSharedState.set(ev, props);
 			contextSharedState.set(`${id}.scrollPosition`, {x, y});
 		}
-	}
+	}, [contextSharedState, props]);
```

Comment:
`props` is always a new object, so `props.id` would be the right dep. It looks missing by accident. :)

Reply:
Right. It could be okay in this line if we define `id` as destructed variable from `props`, but it looks unclear what impact this way may have. I'll look into this.